### PR TITLE
RD-3948 Add POST endpoint in license for submitting a Hubspot Custome…

### PR DESCRIPTION
**Pull request by glukhman**
…r ID in community edition

### CODE REVIEW

File: rest-service/manager_rest/rest/resources_v3_1/license.py [hunk 1]
```diff
@@ -38,6 +45,33 @@ def get(self, license_handler, pagination=None):
...
+        """
+        Store the Customer ID recieved from Hubspot in the Manager.
+        """
+        request_dict = get_json_and_verify_params({
+            'customer_id': {'type': text_type},
+        })
+        sm = get_storage_manager()
+        if len(sm.list(models.License)) > 0:
+            raise ConflictError(
+                'A Customer ID already exists for this manager')
```
Comment (mateumann) [inline rest-service/manager_rest/rest/resources_v3_1/license.py:61]: Maybe more detailed error message would be useful, e.g. what is the customer_id or something like that?

File: rest-service/manager_rest/rest/resources_v3_1/license.py [hunk 2]
```diff
@@ -38,6 +45,34 @@ def get(self, license_handler, pagination=None):
...
+        return sm.put(models.License(customer_id=request_dict['customer_id']))
```
Comment (mateumann) [inline rest-service/manager_rest/rest/resources_v3_1/license.py:62]: is `customer_id` the only interesting thing here?

File: rest-service/manager_rest/rest/resources_v3_1/license.py [hunk 3]
```diff
@@ -38,6 +45,34 @@ def get(self, license_handler, pagination=None):
         return license_handler.list_license()
 
 
+class LicenseCommunity(SecuredResource):
+    @marshal_with(responses_v3.License)
+    @authorize('license_upload')
+    def post(self):
+        """
+        Store the Customer ID recieved from Hubspot in the Manager.
```
Comment (tehasdf) [inline rest-service/manager_rest/rest/resources_v3_1/license.py:53]:
```suggestion
        Store the Customer ID received from Hubspot in the Manager.
```

File: rest-service/manager_rest/rest/resources_v3_1/license.py (hunk 2 above)
Comment (glukhman) [reply rest-service/manager_rest/rest/resources_v3_1/license.py:62]: currently at least

**Review by mateumann — APPROVED** [review]

**Pull request merged**